clients/src/main/java/org/apache/kafka/common/requests/AbstractRequest.java

```diff
@@ -309,6 +309,8 @@ private static AbstractRequest doParseRequest(ApiKeys apiKey, short apiVersion,
                 return LiCombinedControlRequest.parse(buffer, apiVersion);
             case LI_MOVE_CONTROLLER:
                 return LiMoveControllerRequest.parse(buffer, apiVersion);
+            case LI_FEDERATED_TOPIC_CREATE:
+                return LiFederatedTopicCreateRequest.parse(buffer, apiVersion);
```

Comment:
nit: Similarly, per discussion offline today, we should include "znode" in the name to explicitly call out what it is doing.  Otherwise from the name it sounds like it creates a physical topic.